client/utils.ts

```diff
@@ -19,8 +20,21 @@ export function debounce< T, U >( callback: ( ...args: T[] ) => U, timeout: numb
 	};
 }
 
-export function redirectToLaunchpad( siteSlug: string, launchpadFlow: string ) {
-	window.location.replace( `/setup/launchpad?flow=${ launchpadFlow }&siteSlug=${ siteSlug }` );
+export function redirectToLaunchpad(
+	siteSlug: string,
+	launchpadFlow: string,
+	emailVerifiedParam: boolean
+) {
+	let launchpadRedirectionURL = `/setup/launchpad?flow=${ launchpadFlow }&siteSlug=${ siteSlug }`;
+	if ( emailVerifiedParam ) {
+		launchpadRedirectionURL = addQueryArgs(
+			{
+				emailVerified: 1,
+			},
+			launchpadRedirectionURL
+		);
+	}
```

Comment:
`addQueryArgs` automatically omits `undefined` values (but not null).
```suggestion
	let launchpadRedirectionURL = addQueryArgs('/setup/launchpad', {
		flow: launchpadFlow,
		siteSlug,
		emailVerified: emailVerifiedParam ? 1 : undefined
	});
```